starter-code/src/clue.js

```diff
@@ -85,10 +85,77 @@ name: Patio
 */
 
 // Characters Collection
-var charactersArray = [];
+//var charactersArray = [];
 
 // Rooms' Collection
-var roomsArray = [];
+//var roomsArray = [];
 
 // Weapons Collection
-var weaponsArray = [];
+//var weaponsArray = [];
+
+
+
+
+// Characters Collection
+var charactersArray = [
+  
+    {first_name: "Jacob", last_name: "Green", color: "Green", description: "Has a lot of connections", age: "45", image: "https://pbs.twimg.com/profile_images/506787499331428352/65jTv2uC.jpeg", occupation: "Entrepreneur"},
+  
+  {first_name: "Doctor", last_name: "Orchid", color: "White", description: "PhD in plant toxicology. Adopted daughter of Mr. Boddy", age: "26", image: "http://www.radiotimes.com/uploads/images/Original/111967.jpg", occupation: "scientist"},
+  
+  {first_name: "Victor", last_name: "Plum", color: "Purple", description: "Billionare video game designer", age: "22", image: "https://metrouk2.files.wordpress.com/2016/07/professor-plum.jpg", occupation: "Designer"},
+  
+  {first_name: "Kasandra", last_name: "Scarlet", color: "Red", description: "She is an A-list movie star with a dark past", age: "31", image: "https://metrouk2.files.wordpress.com/2016/07/miss-scarlett.jpg", occupation: "Actor"},
+  
+  {first_name: "Eleanor", last_name: "Peacock", color: "Blue", description: "She is from a wealthy family and uses her status and money to earn popularity", age: "36", image: "https://metrouk2.files.wordpress.com/2016/07/mrs-peacock.jpg", occupation: "Socialite"},
+  
+  {first_name: "Jack", last_name: "Mustard", color: "Yellow", description: "He is a former football player who tries to get by on his former glory", age: "62",image: "https://metrouk2.files.wordpress.com/2016/07/colonel-mustard.jpg", occupation: "Retired Football player"}];
+  
+  // Rooms' Collection
+  
+  var roomsArray = [ 
+  {name: "Dinning Room"},
+  {name: "Conservatory"},
+  {name: "Kitchen"},
+  {name: "Study"},
+  {name: "Library"},
+  {name: "Billiard Room"},
+  {name: "Lounge"},
+  {name: "Ballroom"},
+  {name: "Hall"},
+  {name: "Spa"},
+  {name: "Living Room"},
+  {name: "Observatory"},
+  {name: "Theater"},
+  {name: "Guest House"},
+  {name: "Patio"},
+  ];
+  
+  // Weapons Collection
+  var weaponsArray = [
+    {name: "rope", weight: 10},
+  {name: "knife", weight: 8},
+  {name: "candlestick", weight: 2},
+  {name: "dumbbell", weight: 30},
+  {name: "poison", weight: 2},
+  {name: "axe", weight: 15},
+  {name: "bat", weight: 13},
+  {name: "trophy", weight: 25},
+  {name: "pistol", weight: 20},
+  ];
+  
+  
+  function randomSelector(array1){
+    var random = Math.floor(Math.random() * array1.length);
+    return array1[random];
+  }  
+  
+  function pickMistery(){
+    var mysteryEnvelope = [randomSelector(charactersArray), randomSelector(weaponsArray), randomSelector(roomsArray)];
+    return mysteryEnvelope;
+  }
+
+  function revealMistery(x){
```

Comment:
Pick a better name for a parameter in a function, `x` is not a good name for a parameter. You could have gone with this:
```js
function revealMistery(mistery) {
  return mistery[0].first_name + ' ' + mistery[0].last_name + ' killed Mr.Boddy using the ' + mistery[1].name + ' in the ' + mistery[2].name + '!!!!';
}
```